spec/InfiniteIterableValueSpec.php

```diff
@@ -79,20 +83,22 @@ function it_can_flat_map_empty_array()
 
     private function shouldFlatMapBandMembersWith(callable $mapper): void
     {
-        $this->beConstructedWith([
-            [
-                'band' => 'The Beatles',
-                'members' => ['John', 'Paul', 'George', 'Ringo'],
-            ],
-            [
-                'band' => 'Rolling Stones',
-                'members' => ['Mick', 'Keith', 'Ron', 'Charlie'],
-            ],
+        $this->beConstructedWith(
```

Comment:
Please change the formatting back as it was, because it makes the actual changes less visible.